### PR TITLE
Add/update Harvester landing page

### PR DESCRIPTION
**Pull request by btat**
Fixes https://github.com/rancher/rancher-docs/issues/931

## Description

Adds a landing page for the Harvester integration. 

- Existing integration page has been renamed to overview and existing links now point to that
- Add/update canonical links.

### CODE REVIEW

File: docs/integrations-in-rancher/harvester/harvester.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+title: Virtualization on Kubernetes with Harvester
+---
+
+## Harvester
+
+Introduced in Rancher v2.6.1, Harvester is an open-source hyper-converged infrastructure (HCI) software built on Kubernetes. Harvester installs on bare metal servers and provides integrated virtualization and distributed storage capabilities. Although Harvester operates using Kubernetes, it does not require users to know Kubernetes concepts, making it a more user-friendly application.
+
+## Harvester with Rancher
+
+With Rancher Prime and Harvester, IT operators now have access to an enterprise-ready, simple-to-use infrastructure platform that cohesively manages their virtual machines and Kubernetes clusters alongside one another. For more information on the support offering, see the [Support Matrix](https://www.suse.com/suse-harvester/support-matrix/all-supported-versions/harvester-v1-2-0/). With the Rancher Virtualization Management feature, users can import and manage multiple Harvester clusters. Leveraging the Rancher's authentication feature and RBAC control for multi-tenancy support.
```
Comment (martyav) [inline docs/integrations-in-rancher/harvester/harvester.md:11]:
```suggestion
With Rancher Prime and Harvester, your IT operators have an enterprise-ready, simple-to-use infrastructure platform that cohesively manages their virtual machines and Kubernetes clusters alongside one another. 

Rancher's Virtualization Management empowers your team to import and manage multiple Harvester clusters, and leverage Rancher's authentication feature and RBAC control for multi-tenancy support.

See the [Support Matrix](https://www.suse.com/suse-harvester/support-matrix/all-supported-versions/harvester-v1-2-0/) for more details about hardware requirements, supported software, and the Harvester support lifecycle. 
```

Who is the audience for this page? I'm taking it as, people who have the power to get their team to use Rancher and Harvester.

File: docs/integrations-in-rancher/harvester/harvester.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+title: Virtualization on Kubernetes with Harvester
+---
+
+## Harvester
+
+Introduced in Rancher v2.6.1, Harvester is an open-source hyper-converged infrastructure (HCI) software built on Kubernetes. Harvester installs on bare metal servers and provides integrated virtualization and distributed storage capabilities. Although Harvester operates using Kubernetes, it does not require users to know Kubernetes concepts, making it a more user-friendly application.
```
Comment (martyav) [inline docs/integrations-in-rancher/harvester/harvester.md:7]:
```suggestion
Introduced in Rancher v2.6.1, Harvester is an open-source, hyper-converged infrastructure (HCI) software built on Kubernetes. Harvester installs on bare metal servers and provides integrated virtualization and distributed storage capabilities. Although Harvester operates using Kubernetes, it does not require knowledge of Kubernetes concepts, making it more user-friendly.
```

File: docs/integrations-in-rancher/harvester/overview.md [hunk 3]
```diff
@@ -24,7 +24,7 @@ The [Harvester node driver](https://docs.harvesterhci.io/v1.1/rancher/node/node-
 
 Harvester allows `.ISO` images to be uploaded and displayed through the Harvester UI, but this is not supported in the Rancher UI. This is because `.ISO` images usually require additional setup that interferes with a clean deployment (without requiring user intervention), and they are not typically used in cloud environments.
 
-Click [here](../pages-for-subheaders/about-provisioning-drivers.md#node-drivers) for more information on node drivers in Rancher.
+Click [here](../../pages-for-subheaders/about-provisioning-drivers.md#node-drivers) for more information on node drivers in Rancher.
```
Comment (martyav) [inline docs/integrations-in-rancher/harvester/overview.md:27]:
```suggestion
See [Provisioning Drivers](../../pages-for-subheaders/about-provisioning-drivers.md#node-drivers) for more information on node drivers in Rancher.
```

File: versioned_docs/version-2.7/integrations-in-rancher/harvester/harvester.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+---
+title: Virtualization on Kubernetes with Harvester
+---
+
+## Harvester
+
+Introduced in Rancher v2.6.1, Harvester is an open-source hyper-converged infrastructure (HCI) software built on Kubernetes. Harvester installs on bare metal servers and provides integrated virtualization and distributed storage capabilities. Although Harvester operates using Kubernetes, it does not require users to know Kubernetes concepts, making it a more user-friendly application.
```
Comment (martyav) [inline versioned_docs/version-2.7/integrations-in-rancher/harvester/harvester.md:7]:
```suggestion
Introduced in Rancher v2.6.1, Harvester is an open-source, hyper-converged infrastructure (HCI) software built on Kubernetes. Harvester installs on bare metal servers and provides integrated virtualization and distributed storage capabilities. Although Harvester operates using Kubernetes, it does not require knowledge of Kubernetes concepts, making it more user-friendly.
```

File: versioned_docs/version-2.7/integrations-in-rancher/harvester/harvester.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+
+## Harvester with Rancher
+
+With Rancher Prime and Harvester, IT operators now have access to an enterprise-ready, simple-to-use infrastructure platform that cohesively manages their virtual machines and Kubernetes clusters alongside one another. For more information on the support offering, see the [Support Matrix](https://www.suse.com/suse-harvester/support-matrix/all-supported-versions/harvester-v1-2-0/). With the Rancher Virtualization Management feature, users can import and manage multiple Harvester clusters. Leveraging the Rancher's authentication feature and RBAC control for multi-tenancy support.
```
Comment (martyav) [inline versioned_docs/version-2.7/integrations-in-rancher/harvester/harvester.md:11]:
```suggestion
With Rancher Prime and Harvester, your IT operators have an enterprise-ready, simple-to-use infrastructure platform that cohesively manages their virtual machines and Kubernetes clusters alongside one another. 

Rancher's Virtualization Management empowers your team to import and manage multiple Harvester clusters, and leverage Rancher's authentication feature and RBAC control for multi-tenancy support.

See the [Support Matrix](https://www.suse.com/suse-harvester/support-matrix/all-supported-versions/harvester-v1-2-0/) for more details about hardware requirements, supported software, and the Harvester support lifecycle. 
```

File: versioned_docs/version-2.8/integrations-in-rancher/harvester/harvester.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+title: Virtualization on Kubernetes with Harvester
+---
+
+## Harvester
+
+Introduced in Rancher v2.6.1, Harvester is an open-source hyper-converged infrastructure (HCI) software built on Kubernetes. Harvester installs on bare metal servers and provides integrated virtualization and distributed storage capabilities. Although Harvester operates using Kubernetes, it does not require users to know Kubernetes concepts, making it a more user-friendly application.
+
+## Harvester with Rancher
+
+With Rancher Prime and Harvester, IT operators now have access to an enterprise-ready, simple-to-use infrastructure platform that cohesively manages their virtual machines and Kubernetes clusters alongside one another. For more information on the support offering, see the [Support Matrix](https://www.suse.com/suse-harvester/support-matrix/all-supported-versions/harvester-v1-2-0/). With the Rancher Virtualization Management feature, users can import and manage multiple Harvester clusters. Leveraging the Rancher's authentication feature and RBAC control for multi-tenancy support.
```
Comment (martyav) [inline versioned_docs/version-2.8/integrations-in-rancher/harvester/harvester.md:11]:
```suggestion
With Rancher Prime and Harvester, your IT operators have an enterprise-ready, simple-to-use infrastructure platform that cohesively manages their virtual machines and Kubernetes clusters alongside one another.

Rancher's Virtualization Management empowers your team to import and manage multiple Harvester clusters, and leverage Rancher's authentication feature and RBAC control for multi-tenancy support.

See the [Support Matrix](https://www.suse.com/suse-harvester/support-matrix/all-supported-versions/harvester-v1-2-0/) for more details about hardware requirements, supported software, and the Harvester support lifecycle. 
```

File: versioned_docs/version-2.8/integrations-in-rancher/harvester/harvester.md [hunk 7]
```diff
@@ -0,0 +1,11 @@
+---
+title: Virtualization on Kubernetes with Harvester
+---
+
+## Harvester
+
+Introduced in Rancher v2.6.1, Harvester is an open-source hyper-converged infrastructure (HCI) software built on Kubernetes. Harvester installs on bare metal servers and provides integrated virtualization and distributed storage capabilities. Although Harvester operates using Kubernetes, it does not require users to know Kubernetes concepts, making it a more user-friendly application.
```
Comment (martyav) [inline versioned_docs/version-2.8/integrations-in-rancher/harvester/harvester.md:7]:
```suggestion
Introduced in Rancher v2.6.1, Harvester is an open-source, hyper-converged infrastructure (HCI) software built on Kubernetes. Harvester installs on bare metal servers and provides integrated virtualization and distributed storage capabilities. Although Harvester operates using Kubernetes, it does not require knowledge of Kubernetes concepts, making it more user-friendly.
```

File: versioned_docs/version-2.7/integrations-in-rancher/harvester/overview.md [hunk 8]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Harvester Integration
+title: Overview
 ---
 
 Introduced in Rancher v2.6.1, [Harvester](https://docs.harvesterhci.io/) is an open-source hyper-converged infrastructure (HCI) software built on Kubernetes. Harvester installs on bare metal servers and provides integrated virtualization and distributed storage capabilities. Although Harvester operates using Kubernetes, it does not require users to know Kubernetes concepts, making it a more user-friendly application.
```
Comment (martyav) [inline versioned_docs/version-2.7/integrations-in-rancher/harvester/overview.md:5]:
```suggestion
Introduced in Rancher v2.6.1, Harvester is an open-source, hyper-converged infrastructure (HCI) software built on Kubernetes. Harvester installs on bare metal servers and provides integrated virtualization and distributed storage capabilities. Although Harvester operates using Kubernetes, it does not require knowledge of Kubernetes concepts, making it more user-friendly.
```

**Review by martyav — DISMISSED** [review]

File: docs/integrations-in-rancher/harvester/harvester.md (hunk 1 above)
Comment (btat) [reply docs/integrations-in-rancher/harvester/harvester.md:11]: @CamrynCarter Do we have a specific target audience for the various landing pages?

**Comment by btat** [comment]
Per discussion with Guna, we're proceeding to merge so stakeholders are able to preview the update. We'll continue to improve/iterate post-merge.

**Pull request merged**